streampipes-connect-management/src/main/java/org/apache/streampipes/connect/management/management/GuessManagement.java

```diff
@@ -57,10 +56,8 @@ public GuessSchema guessSchema(AdapterDescription adapterDescription)
     var objectMapper = JacksonSerializer.getObjectMapper();
     var description = objectMapper.writeValueAsString(adapterDescription);
     logger.info("Guess schema at: " + workerUrl);
-    Response requestResponse = Request.Post(workerUrl)
-        .bodyString(description, ContentType.APPLICATION_JSON)
-        .connectTimeout(1000)
-        .socketTimeout(100000)
+    Response requestResponse = ExtensionServiceExecutions
+        .extServicePostRequest(workerUrl, null, description)
```

Comment:
Is it possible that this `null` value causes an exception? Because in the `AuthTokenUtils.getAuthToken` the `ResourceId` is checked for null and then throws an exception